tsconfig.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
We'll add options later like enabling strict mode. For now, we only need the file to exist